src/browser/api/browser_view.ts

```diff
@@ -35,31 +38,46 @@ export async function create(options: BrowserViewOpts) {
         setBounds(ofView, options.bounds);
     }
 }
-
+export function hide(ofView: OfView) {
+    const win = getElectronBrowserWindow(ofView.target);
+    win.removeBrowserView(ofView.view);
+}
+export function show(ofView: OfView) {
+    const win = getElectronBrowserWindow(ofView.target);
+    win.addBrowserView(ofView.view);
+}
 export async function attach(ofView: OfView, toIdentity: Identity) {
     const {view} = ofView;
     if (view) {
+        if (ofView.target.name !== toIdentity.name) {
+            const oldWin = getWindowByUuidName(ofView.target.uuid, ofView.target.name);
+            if (oldWin) {
+                const oldwinMap = windowCloseListenerMap.get(oldWin);
+                if (oldwinMap) {
+                    const listener = oldwinMap.get(ofView);
+                    of_events.removeListener(route.window('closed', ofView.target.uuid, ofView.target.name), listener);
```

Comment:
since all the other checks exist.... I think removeListener will throw if listener is undefined.

Reply:
has this been addressed ?